docs/DeveloperGuide.md

```diff
@@ -118,17 +118,19 @@ How the parsing works:
 ### Model component
 **API** : [`Model.java`](https://github.com/AY2324S1-CS2103T-T13-0/tp/blob/master/src/main/java/seedu/address/model/Model.java)
 
-<img src="diagrams/ModelClassDiagram.png" width="450" />
+<puml src="diagrams/ModelClassDiagram.puml" width="450" />
 
+<puml src="diagrams/ModuleClassDiagram.puml" width="450" />
 
 The `Model` component,
 
 * stores the module plan of each semester i.e., all `ModulePlanSemester` in one `ModulePlanSemesterList`.
-* * stores the module plan data i.e., all `Module` objects (which are contained in a `UniqueModuleList` object).
+* stores the module plan data i.e., all `Module` objects (which are contained in a `UniqueModuleList` object).
 * each module plan of a semester contains a `Semester` and a `Year` for identification and stores a `UniqueModuleList`.
 * stores the currently 'selected' `Module` objects (e.g., results of a search query) as a separate _filtered_ list which is exposed to outsiders as an unmodifiable `ObservableList<Module>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
```

Comment:
This doesn't seem to still be relevant after the removal of the filteredPersons field. Can this line be deleted?